services/app-web/src/pages/StateSubmission/RateDetails/V2/RateDetailsV2.tsx

```diff
@@ -298,7 +298,12 @@ const RateDetailsV2 = ({
         if (rateErrors && Array.isArray(rateErrors)) {
             rateErrors.forEach((rateError, index) => {
                 if (!rateError) return
-
+                if (
+                    Object.keys(rateError).includes('ratePreviouslySubmitted')
+                ) {
+                    return (errorObject['ratePreviouslySubmitted'] =
```

Comment:
a little confused why this is needed and its not handled by the schema. can you add a comment if this stays here. 🙏